vita3k/modules/SceSysmem/SceSysmem.cpp

```diff
@@ -55,6 +55,10 @@ struct SceKernelFreeMemorySizeInfo {
     int size_phycont; //!< Free memory size for USER_MAIN_PHYCONT_*_RW memory
 };
 
+struct KernelMemBlock : SceKernelMemBlockInfo {
+    char name[KERNELOBJECT_MAX_NAME_LENGTH + 1];
+};
+
 typedef std::shared_ptr<SceKernelMemBlockInfo> SceKernelMemBlockInfoPtr;
 typedef std::map<SceUID, SceKernelMemBlockInfoPtr> Blocks;
```

Comment:
```suggestion
typedef std::shared_ptr<KernelMemBlock> KernelMemBlockPtr;
typedef std::map<SceUID, KernelMemBlockPtr> Blocks;
```

Reply:
Done, i have also updated so std::;find no need reinterpret now and works fine 👌